README.rst

```diff
@@ -56,11 +56,13 @@ Links
 
 * Feedback and ideas: http://mail.python.org/mailman/listinfo/code-quality
 
-* Cyclomatic complexity: http://en.wikipedia.org/wiki/Cyclomatic_complexity.
+* Cyclomatic complexity: http://en.wikipedia.org/wiki/Cyclomatic_complexity
 
 * Ned Batchelder's script:
   http://nedbatchelder.com/blog/200803/python_code_complexity_microtool.html
 
+* McCabe complexity: http://en.wikipedia.org/wiki/Cyclomatic_complexity
```

Comment:
Hmm, I'm not sure how this ended up here. My intention was to make the first mention of "McCabe complexity" at the start of the README a link.

@sigmavirus24 - I can remove this duplicate link from here and put it on that first mention if you want.